packages/cli/src/databases/config.ts

```diff
@@ -51,28 +47,63 @@ export const getOptionOverrides = (dbType: 'postgresdb' | 'mysqldb') => ({
 	password: config.getEnv(`database.${dbType}.password`),
 });
 
-export const getSqliteConnectionOptions = (): SqliteConnectionOptions => ({
+const getSqliteConnectionOptions = (): SqliteConnectionOptions => ({
 	type: 'sqlite',
-	...getDBConnectionOptions('sqlite'),
+	...getCommonOptions(),
+	database: path.resolve(
+		Container.get(InstanceSettings).n8nFolder,
+		config.getEnv('database.sqlite.database'),
+	),
+	enableWAL: config.getEnv('database.sqlite.enableWAL'),
 	migrations: sqliteMigrations,
 });
 
-export const getPostgresConnectionOptions = (): PostgresConnectionOptions => ({
-	type: 'postgres',
-	...getDBConnectionOptions('postgresdb'),
-	schema: config.getEnv('database.postgresdb.schema'),
-	poolSize: config.getEnv('database.postgresdb.poolSize'),
-	migrations: postgresMigrations,
-});
+const getPostgresConnectionOptions = (): PostgresConnectionOptions => {
+	const sslCa = config.getEnv('database.postgresdb.ssl.ca');
+	const sslCert = config.getEnv('database.postgresdb.ssl.cert');
+	const sslKey = config.getEnv('database.postgresdb.ssl.key');
+	const sslRejectUnauthorized = config.getEnv('database.postgresdb.ssl.rejectUnauthorized');
 
-export const getMysqlConnectionOptions = (): MysqlConnectionOptions => ({
-	type: 'mysql',
-	...getDBConnectionOptions('mysqldb'),
-	migrations: mysqlMigrations,
-});
+	let ssl: TlsOptions | boolean = config.getEnv('database.postgresdb.ssl.enabled');
+	if (sslCa !== '' || sslCert !== '' || sslKey !== '' || !sslRejectUnauthorized) {
+		ssl = {
+			ca: sslCa || undefined,
+			cert: sslCert || undefined,
+			key: sslKey || undefined,
+			rejectUnauthorized: sslRejectUnauthorized,
+		};
+	}
+
+	return {
+		type: 'postgres',
+		...getCommonOptions(),
+		...getOptionOverrides('postgresdb'),
+		schema: config.getEnv('database.postgresdb.schema'),
+		poolSize: config.getEnv('database.postgresdb.poolSize'),
+		migrations: postgresMigrations,
```

Comment:
Should these be imported dynamically for each DB? Not sure if there is much perf impact tho

Reply:
likely not going to make much difference in performance or memory usage.